README.md

```diff
@@ -26,6 +26,7 @@ Supported outlets:
 |*Vox*|`vox`|✔️|||
 |*Wall Street Journal*|`wsj`|✔️||✔️|
 |*Washington Post*|`wp`|✔️|✔️||
+|*Mc Kinsey*|`mck`|✔️|✔️|✔️|
```

Comment:
Please alphabetize between Los Angeles Times and New York Times. Also, again this is idiosyncratic, but can we use `The McKinsey Crossword` here and where I note below? That appears to be the official branding.

Reply:
done